e0works/MyDiary-front
# #162312873 A user should be able see dashboard page after sign up or login

#### What does this PR do?

#### Description of Task to be completed?
- add a sidebar
- add header
- add menu on sidebar to toggle main view
- add create entry modal

#### How should this be manually tested?
- start the app
- login, you should be redirected to dashboard page
- click on menu item on the sidebar to view different section of the dashboard page

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#162312873](https://www.pivotaltracker.com/story/show/162312873, "user dashboard page")

#### Screenshots (if appropriate)
<img width="1680" alt="screen shot 2018-11-29 at 7 36 08 pm" src="https://user-images.githubusercontent.com/24890605/49243624-0ecf8280-f40e-11e8-8645-9b4ed93e29bb.png">


#### Questions:
NA
